_posts/2021-04-20-First-Year.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: post
+title: "First year of the Fortran website"
+category: newsletter
```

Comment:
We have to change the category...

Reply:
Can it be just "post"?

Reply:
The 'category' isn't displayed anywhere on site and it looks like the site is currently hard-coded to only display posts in the 'newsletter' category, which needs fixing.

https://github.com/fortran-lang/fortran-lang.org/blob/a80ff0e335b18500e775cd0fd62cbbded78a9dfd/news/index.html#L12

Reply:
I see. I changed it to "post", but that would not show up, so I changed it back to "newsletter".